features/step_definitions/logout_steps.rb

```diff
@@ -0,0 +1,29 @@
+Given(/^that I am logged in as "([^"]*)"$/) do |name|
+  user = User.create(name: name)
+  login_as(user, scope: :user)
+end
```

Comment:
Ok so what happens if we want to re-use this step, but our users already exists? We should break these into two things. `Given the following users exist` or just `Given there is a user with name "Test-User"` and then the login. That way we can use *both* of those actions independently in future tests.